File: 1/1.cpp
/*!***********************
 * @brief:  leetcode-1-两数之和
 * @date:   2019/5/9
 * @envir:  gcc 7.4.0
 * @author: luhao
 * ***********************/

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int l = nums.size();
        for (int i=0; i<l-1; i++){
            for (int j=i+1; j<l; j++){
                if (nums[i]+nums[j]==target){
                    return vector<int>{i,j};
                }       
            }         
        }
        return vector<int>{};
    }
};

File: 1/1.py
# @brief:  leetcode-1-两数之和
# @envir:  python 2.7
# @date:   2019/5/9
# @author: luhao

class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        ans = []
        for i in range(len(nums)-1):
            for j in range(i+1,len(nums)):
                if nums[i]+nums[j]==target:
                    return [i,j]
        
